presto-product-tests/conf/docker/multinode/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '2'
 services:
   hadoop-master:
     hostname: hadoop-master
-    image: 'teradatalabs/cdh5-hive:2.6.0-cdh5.6.0-sshd-4e54a19'
+    image: 'teradatalabs/cdh5-hive:2'
```

Comment:
I think it is better to use `commit id` as a tag. So it is easier to understand what some particular image include.